.github/workflows/platform-content-sync.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync content with platform
+
+on:
+  pull_request:
+    types:
+      - closed
+    paths:
+      - '**.mdx'
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: carbon-design-system/carbon-platform
+          ref: main
```

Comment:
Might not need the checkout/clone here since this doesn't look to hit the repo code at all.

Reply:
@jdharvey-ibm do you have alternative suggestion?